src/Agent.Worker/Handlers/ProcessHandler/ProcessHandler.cs

```diff
@@ -203,6 +203,11 @@ public async Task RunAsync()
             // Invoke the process.
             ExecutionContext.Debug($"{cmdExe} {cmdExeArgs}");
             ExecutionContext.Command($"{cmdExeArgs}");
+
+            var sigintTimeout = TimeSpan.FromMilliseconds(AgentKnobs.ProccessSigintTimeout.GetValue(ExecutionContext).AsInt());
```

Comment:
Are these timeout values always provided? When the default ones will be used?

Reply:
Knob sources have the following priority:
1. Runtime source (from pipelines variables)
2. Environment source (from environment variables)
3. Built-in source (hardcoded in a knob definition)

If neither runtime nor environment sources are provided, the agent will use built-in source